.github/workflows/python-dev.yml

```diff
@@ -40,12 +48,17 @@ jobs:
         python-version: '3.10-dev'
 
     - name: Install dependencies
+      shell: bash
       run: |
         python -m pip install --upgrade pip setuptools wheel
-        pip install -i https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
-        pip install git+https://github.com/pytest-dev/pytest.git
+        echo $NUMPY_WHEELS_AVAILABLE
```

Comment:
Where do we set this variable?

Reply:
https://github.com/pandas-dev/pandas/pull/43772/files#diff-f6f33d5fc5cfd739f0d6efd63ea9dff886abe35cdfff5672e9a863ec8da8e72cR34